### ISSUE TITLE
[FEATURE] Request access button on dataset landing page

### ISSUE THREAD

**Issue by DorienHuijser** [UtrechtUniversity/yoda-ansible#378]
#### Is your feature request related to a problem? Please describe.

When a dataset is published under restricted access, there is now no way for external users to find out  _how_ to gain access. The only contact information on the landing page is the email address of RDM Support, and we actually don't usually know who is responsible for a dataset, or what the terms for access are. 

#### Describe the solution you'd like
A button "Request access" / "Contact data owner" similar to the implementation in DataverseNL. This would preferably lead to a form (like in DataverseNL), but could also just be a link like `mailto::emailaddress@institute.nl`. If the latter is implemented, it should be made clear in the metadata form that this email address will be made publicly available.

#### Describe alternatives you've considered
If issue #376 is implemented, contact information could also be provided in the license.txt and this issue is not so pressing anymore.

#### Additional context
This question was raised by one of the faculty data stewards at UU in the context of an already published restricted access dataset.

**Comment by Danny-dK** [comment]
I could imagine adding a metadata field in the Yoda template that becomes mandatory to fill in once 'restricted access' is chosen. Something like 'contact email for data access'?

**Comment by peer35** [comment]
We do recognize the need for a "Data Request" workflow at the VU, but I feel just adding an email address will not work. Who will maintain this contact information? I imagine the contact will be a researcher, what happens if that person has left the institution? I actually prefer a more general address like RDM support, they can then send the request on. 

Alternatively a system where the group managers and the datamanagers receive the request might be helpful.

**Comment by Danny-dK** [comment]
Yes, during evaluation of a submission I always mention to the researcher that they need to provide a more sustainable email address. This could for example be the secretariat of the chairgroup or some other science group address. That doesn't mean of course that a chairgroup is completely sustainable as they may merge or end, but it is more sustainable then an personal address. I go back and forth on RDM support being a contact point. Depending on the university that may have many faculties, that might not be the most fun activity for the person(s) of RDM support. 

It would be nice if the publication 'request access' button indeed has a link to the category it belonged to and that the Yoda data manager role of that category is than notified. The actual person behind that role may change over time, but it is unlikely that the role in Yoda will change.

**Comment by MKoperdraad** [comment]
We have started work on a redesign of the datarequest module, including making it more generic. One of the requirements is the ability to use it for requesting restricted/sensitive data by means of a request button on landing page.
